axum-extra/src/extract/with_rejection.rs

```diff
@@ -137,6 +137,27 @@ where
     }
 }
 
+#[cfg(feature = "typed-routing")]
+const _: () = {
```

Comment:
Why do it this way? I find the `const _` trick a bit weird.

Reply:
Just so you don't need to put `#[cfg(feature = "typed-routing")]` on the imports as well (unless you want warnings) or use fully qualified paths.

Reply:
I think I'd prefer either of those solutions 😅

cc https://github.com/rust-lang/rust/issues/65516 ;)

Reply:
I hope that doesn't land 😅 I think its pretty useful to be able to define stuff everywhere though I do see its a nightmare from and IDE point of view.

But I've fixed it in c1b27565976b0